src/sections/Blog/BlogPostSignOff/index.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import {Link, graphql, useStaticQuery} from "gatsby";
+import slugify from "../../../utils/slugify";
+import BlogPostSignOffWrapper from "./blogpost-signoff.styles";
+
+const authorField = (author, isSlugAvailable) => {
```

Comment:
Since this is also being used in `PageHeader` can you create a this function in `utils` can call the file `authorValidator` (or something similar) and use that both here and in PageHeader

Reply:
Based on the slack conversation, moving ahead with this for now